python/lsst/ts/Dome/dome_csc.py

```diff
@@ -361,7 +362,45 @@ async def config(self, _data):
             The contents of this parameter will be defined soon.
         """
```

Comment:
"will be defined soon"? Can you define it now?

Reply:
To be honest, I haven't got a clue how configuration parameters will be passed on from TCS to DCS so the answer to your question is "no".